functions/README.md

```diff
@@ -0,0 +1,67 @@
+# SETUP CLOUD FUNCTIONS
+
+If you haven't initialized your firebase project, please start with setup
+stated in main directory README. This is only cloud functions guide.
+
+To deploy your functions, use
+
+```bash
+firebase deploy --only functions
```

Comment:
> It needs npm install -g firebase-tools which you are recommending below (and it was not in the "setup
stated in main directory README"). How about moving npm install -g firebase-tools to the ~beginning of this file, or to main directory README?

Ok, I'll move this part to main project README. But I'll create another PR for that since it's not assiociated with groups.

> Moreover, for the whole project to work, Cloud Functions need to be deployed as described here, but it's not referenced from the main directory README file. Please do reference it from there, so that one doesn't have to explore the repository folder-by-folder to find all the pieces that need to be deployed.

In the main directory there is `firebase deploy` which should deploy also functions.

Reply:
I created additional PR for that.